web/libs/editor/src/core/__tests__/ConfigValidator.test.js

```diff
@@ -4,42 +4,41 @@ import '../../tags/object/Image';
 import '../../tags/object/RichText';
 import '../../tags/control/RectangleLabels';
 import '../../tags/control/Label';
+import '../../tags/control/Choices';
+import '../../tags/control/Choice';
+import '../../tags/visual/Header';
+import { ConfigValidator } from "../DataValidator/ConfigValidator";
 
-// IMPORTANT NOTE
-// all tests are disabled because they wait `validation` in the root component
-// and this doesn't happen, may be that was some old behavior
-// @todo fix this
+function createStore(utterances) {
+  return { task: { dataObj: { utterances } } };
```

Comment:
A little worrying that this task data is different from what actual configs in tests expect. Do they all fail just with data validation and not a config one?

Reply:
This validation is validating just the data, not the config. So doesn't matter which config we are using, if you write a test with everything good, it will return a empty array without error

Reply:
sorry, didn't get it... this validation specifically validates config. what am I missing?